### PR TITLE
Bugfix/102/1 write unit test for subjects page

### PR DESCRIPTION
**Pull request by holus-bolus**
Created unit tests for the Subjects page

image

### CODE REVIEW

**Review by VictoriaRodik — APPROVED** [review]

**Review by StasRusnak1 — APPROVED** [review]

**Review by Wiret02 — APPROVED** [review]

**Review by iaremchuk — APPROVED** [review]
Just a reminder to delete comments and change 'test' to 'it'.

**Comment by holus-bolus** [comment]
> Just a reminder to delete comments and change 'test' to 'it'.

What's the problem with the comment? Couldn't we comment on our code?

File: tests/unit/pages/subjects/Subjects.spec.jsx
Comment (vlad-khrychov) [inline tests/unit/pages/subjects/Subjects.spec.jsx:1]: This test should be easier if you mock `PageWrapper` component.

**Review by vlad-khrychov — CHANGES_REQUESTED** [review]
Again. If i understand i need to look only at `tests/unit/pages/subjects/Subjects.spec.jsx` but i see a lot of files in this PR.

File: tests/unit/pages/subjects/Subjects.spec.jsx
Comment (holus-bolus) [reply tests/unit/pages/subjects/Subjects.spec.jsx:1]: @vlad-khrychov added the PageWrapper mock

**Pull request closed without merging**